elements/pfe-navigation/src/_variables-mixins.scss

```diff
@@ -0,0 +1,254 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: navigation;
+
+$menu-height: 72px;
+
+$gutter: 32px;
+
+$breakpoints: (
+  sm: 576px,
+  md: 768px,
+  lg: 992px,
+  xl: 1200px,
+  2xl: 1368px,
+  // The below should match an above breakpoint value
+  // Where the icons on the right appear
+  'secondary-links-expand': 768px,
+
+  // Default value for when the nav expands to desktop mode
+  'nav-expand': 1200px,
+);
+
+/// Breakpoint helper specific to pfe-navigation
+/// @param {string} $breakpoint Name of a breakpoint
+/// @see $breakpoints
+@mixin breakpoint($breakpoint) {
+  @media (min-width: #{map-get($breakpoints, $breakpoint)}) {
+    @content;
+  }
+}
+
+/// JS triggered 'breakpoints' if there isn't room for a section a class is added to collapse it
+/// Styles in collapse should undo _desktop layout changes_ that the related breakpoint makes.
+/// @example
+///   pfe-navigation {
+///     display: block;
+///     margin: 10px 0 0;
+///     @include breakpoint('nav-expand') {
+///       display: flex;
+///       margin: 20px 40px;
+///     }
+///     @include collapse(nav) {
+///       display: block;
+///       margin: 10px 0 0;
+///     }
+///   }
+/// @param {string} $section Can be 'nav' or 'secondary-links'
+@mixin collapse($section) {
+  @if $section == 'nav' {
```

Comment:
This seems like an abstraction that could cause difficulty if someone is trying to fix a bug in the main file.